platform/core/src/classes/MetadataProvider.ts

```diff
@@ -506,11 +477,54 @@ const metadataProvider = new MetadataProvider();
 
 export default metadataProvider;
 
+const WADO_IMAGE_LOADER = {
```

Comment:
what is the benefit of moving this down here?

Reply:
Primarily that the function size doesn't keep on growing - there are a hundred plus modules in DICOM, and having them all returned in a single function is very ugly.  This breaks it up so that they are separate functions rather than keeping it all in a single gigantic function.
Also allows adding more modules after the fact, based on having the instance, rather than needing to register them with the CS3D metadata provider which doesn't assume anything about having instances.